producer-rest-core/src/main/scala/za/co/absa/spline/producer/rest/controller/ExecutionEventsController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.producer.rest.controller
+
+import io.swagger.annotations.{Api, ApiOperation, ApiResponse, ApiResponses}
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation._
+import za.co.absa.spline.producer.rest.model.ExecutionEvent
+import za.co.absa.spline.producer.service.repo.ExecutionProducerRepository
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Controller
+@Api(tags = Array("execution"))
+class ExecutionEventsController @Autowired()(
+  val repo: ExecutionProducerRepository) {
+
+  import ExecutionContext.Implicits.global
+
+  @PostMapping(Array("/execution-events"))
+  @ApiOperation(
+    value = "Save Execution Events",
+    notes =
+      """
+        Saves a list of Execution Events.
+
+        RequestBody format :
+
+        Array[ExecutionEvent]
+        [
+          // Reference to the Executionplan Id that was triggered
+          planId: UUID
```

Comment:
Just wondering, why not have commas at the end of the line when separating fields in the examples?

Reply:
Yep, I think the format is strange anyway. Will change it.